home-service/src/test/java/com/softserveinc/ita/homeproject/homeservice/service/impl/PollServiceImplTest.java

```diff
@@ -110,10 +110,19 @@ void createPollWithDeletedHouseTest() {
         PollDto pollDto = new PollDto();
         pollDto.setStatus(PollStatusDto.DRAFT);
         pollDto.setPolledHouses(Collections.singletonList(houseDto));
+        pollDto.setCreationDate(LocalDateTime.now());
         when(cooperationRepository.findById(anyLong())).thenReturn(Optional.of(cooperation));
         assertThrows(NotFoundHomeException.class, () -> pollService.create(1L, pollDto));
     }
 
+    @Test
+    void createPollWithCreationDateInPastTest(){
+        PollDto poll = new PollDto();
+        poll.setStatus(PollStatusDto.DRAFT);
+        poll.setCreationDate(LocalDateTime.now().minusDays(1));
+        assertThrows(BadRequestHomeException.class, () -> pollService.create(1L, poll));
```

Comment:
Specify your exception checking with massage validation. For this you can use assertThatExceptionOfType() form assertj library